bridge-token-factory/tests/token_transfer.rs

```diff
@@ -125,9 +142,28 @@ fn run_view_function(
     remove_quotes(&mut res)
 }
 
+fn get_fee_amount(
+    amount: u128,
+    fee_percentage: u128,
+    lower_bound: u128,
+    upper_bound: u128,
+) -> u128 {
+    const FEE_DECIMAL_PRECISION: u128 = 100_00_00;
+    let mut fee_amount = (amount * fee_percentage) / FEE_DECIMAL_PRECISION;
+    if fee_amount < lower_bound {
+        fee_amount = lower_bound;
+        return fee_amount;
+    } else if fee_amount > upper_bound {
+        fee_amount = upper_bound;
+        return fee_amount;
+    } else {
+        return fee_amount;
+    }
+}
+
 #[test]
-fn test_eth_token_transfer() {
-    let (alice, factory, worker) = create_contract();
+fn test_token_transfer_with_deposit_and_withdraw_fee() {
+    let (alice, factory, _, worker) = create_contract();
```

Comment:
Please don't remove the already existing test. Fee functionality is optional and thus should have all old tests to pass without any issues. If there's a strong reason for it not to pass - please describe it accordingly.

Reply:
This was done accidentally, added it back :)